### PR TITLE
Implement mute stream with simple tests

### PR DESCRIPTION
**Pull request by SecondeJK**
This PR provides functionality for the forceMute feature added to the API.

Link to JIRA ticket:
https://jira.vonage.com/browse/DEVX-5444

## Description
Contains basic tests for typehints and a stock 200 response. The live response from the API is unknown at this point and will be emulated as soon as it is available. Two new methods have been added to both the OpenTok developer facing interface (or more accurate client wrapper) and the underlying guzzle client wrapper.

## Motivation and Context
Ongoing development to roll out new API features, this feature is GA.

## How Has This Been Tested?
Three new tests have been added to the two new methods, the full test suite has been run to pass.

## Types of changes

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/OpenTok/OpenTok.php [hunk 1]
```diff
@@ -478,6 +478,58 @@ public function forceDisconnect($sessionId, $connectionId)
...
+    /**
+     * Mute a stream in an OpenTok session
+     *
+     * @param string $sessionId The OpenTok session ID that the client is connected to
+     *
+     * @param string $streamId The OpenTok streamId to be muted in the session
+     *
+     * @return bool
+     */
+    public function muteStreamInSession(string $sessionId, string $streamId): bool
```
Comment (jeffswartz) [inline src/OpenTok/OpenTok.php:490]: This does not match the proposed API (`OpenTok.forceMuteStream()`. See https://github.com/opentok/api-review/tree/master/mute-all-api.

File: src/OpenTok/OpenTok.php [hunk 2]
```diff
@@ -478,6 +478,58 @@ public function forceDisconnect($sessionId, $connectionId)
...
+    /**
+     * Mute multiple streams in an OpenTok session
+     *
+     * @param string $sessionId The OpenTok session ID that the client is connected to.
+     *
+     * @param array<string> $excludeStreams An array of valid OpenTok stream IDs to mute
+     *
+     * @return bool
+     */
+    public function muteStreamsInSession(string $sessionId, array $excludeStreams): bool
```
Comment (jeffswartz) [inline src/OpenTok/OpenTok.php:516]: This does not match the proposed API (`OpenTok.forceMuteAll()`. See https://github.com/opentok/api-review/tree/master/mute-all-api.

**Review by jeffswartz — CHANGES_REQUESTED** [review]

**Comment by SecondeJK** [comment]
@jeffswartz The requested changes have been made but I can't resolve it - is this something you can do?

**Review by jeffswartz — DISMISSED** [review]

File: src/OpenTok/OpenTok.php [hunk 3]
```diff
@@ -478,6 +478,62 @@ public function forceDisconnect($sessionId, $connectionId)
...
+     *
+     * @param string $sessionId The OpenTok session ID.
+     *
+     * @param array<string> $options This array defines options and includes the following keys:
+     *
+     * <ul>
+     *    <li><code>'excludedStreamIds'</code> (array, optional) &mdash; An array of stram IDs
+     *    corresponding to streams that should not be muted. This is an optional property.
+     *    If you omit this property, all streams in the session will be muted.</li>
+     *
```
Comment (superchilled) [inline src/OpenTok/OpenTok.php:516]: The property name for the `POST` data expected by the [API endpoint](https://tokbox.com/developer/rest/#force_mute_session) is `excludedStreams`, not `excludedStreamIds`.

File: src/OpenTok/Util/Validators.php [hunk 4]
```diff
@@ -30,6 +30,29 @@ public static function validateApiKey($apiKey)
...
+            if (isset($options[$optionName])) {
+                if (getType($options[$optionName]) !== $optionType) {
+                    throw new InvalidArgumentException('Invalid type given in options for: ' . $options[$optionName]);
+                }
+            }
+        }
+
+        if (isset($options['excludedStreamIds'])) {
+            foreach ($options['excludedStreamIds'] as $streamId) {
+                self::validateStreamId($streamId);
```
Comment (superchilled) [inline src/OpenTok/Util/Validators.php:51]: The property name for the `POST` data expected by the [API endpoint](https://tokbox.com/developer/rest/#force_mute_session) is `excludedStreams`, not `excludedStreamIds`.

**Review by superchilled — CHANGES_REQUESTED** [review]
Mostly looks good, though I think one of the the property names in the data payload for `forceMuteAll` is incorrect.

**Review by jeffswartz — DISMISSED** [review]

**Review by jeffswartz — APPROVED** [review]

**Review by superchilled — APPROVED** [review]

**Pull request merged**